### PR TITLE
Release 0.3.3

### LINKED ISSUE #283: Release Version 0.3.3
@allenh1 Would it be possible for you to release a new version of **superflore** now? Or do you want to wait for #282 to be completed?

### PR DESCRIPTION
**Pull request by allenh1**
Signed-off-by: Hunter L. Allen <hunter.allen@ghostrobotics.io>

@shr-project / @herb-kuta-lge  I need to generate the changelog, but this is step one... I'll set it up on my laptop and see how it goes.

---

Closes #283

### CODE REVIEW

**Comment by allenh1** [comment]
@herb-kuta-lge okay, updated changelog. PTAL

**Review by herb-kuta-lge — DISMISSED** [review]

**Comment by tfoote** [comment]
Travis is basically only available for paying clients now. We have generally switched everything over to GIthub Actions which wer running on Travis.

**Comment by herb-kuta-lge** [comment]
I see there's both `master` and `main` branches. Is the intent that they always be coincident? Currently, they're not.

**Comment by tfoote** [comment]
As repos transition to main from master I've found using this github action to automatically mirror it useful. Example in rocker: https://github.com/osrf/rocker/blob/main/.github/workflows/mirror-main-to-master.yaml

**Comment by allenh1** [comment]
@herb-kuta-lge / @tfoote PTAL

@nuclearsandwich thanks for getting the actions working for us!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+[Full Changelog](https://github.com/ros-infrastructure/superflore/compare/v0.3.2...v0.3.3)
+
+**Closed issues:**
+
+- License parsing doesn't parse e.g. LGPLv3 correctly [\#271](https://github.com/ros-infrastructure/superflore/issues/271)
+
+**Merged pull requests:**
+
+- Fix parsing of \<exports\>\<build\_type\> with conditionals [\#281](https://github.com/ros-infrastructure/superflore/pull/281) ([shr-project](https://github.com/shr-project))
```
Comment (herb-kuta-lge) [inline CHANGELOG.md:13]: Should there be an entry for https://github.com/ros-infrastructure/superflore/pull/288 ?

**Review by herb-kuta-lge — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (allenh1) [reply CHANGELOG.md:13]: Yep. Sorry.

**Review by herb-kuta-lge — APPROVED** [review]

**Review by shr-project — APPROVED** [review]

**Review by tfoote — APPROVED** [review]

**Pull request merged**